<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cache Space</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>671</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Statistic</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cache Size:</string>
      </property>
     </widget>
     <widget class="QSlider" name="stCacheSize">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Algorithms:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="stLRU">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>LRU</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="stMRU">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>MRU</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Queue Size:</string>
      </property>
     </widget>
     <widget class="QSlider" name="stQueueSize">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="stCountAtt">
      <property name="geometry">
       <rect>
        <x>176</x>
        <y>140</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Attempts number:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stCountStat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Count</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="stRR">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>RR</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="stCacheOutput">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="stQueueOutput">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="stForEachAlg">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>60</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>For Each</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="stReport">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>651</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="stClearLog">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>310</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Log</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Graph</string>
     </attribute>
     <widget class="QCheckBox" name="grForEach">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>110</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>For each</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="grLRU">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>20</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>LRU</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="grMRU">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>50</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>MRU</string>
      </property>
     </widget>
     <widget class="QPushButton" name="grDrawPlots">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>310</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Draw</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="grapgArea">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>481</width>
        <height>331</height>
       </rect>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="CrossPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="sceneRect">
       <rectf>
        <x>0.000000000000000</x>
        <y>0.000000000000000</y>
        <width>400.000000000000000</width>
        <height>270.000000000000000</height>
       </rectf>
      </property>
     </widget>
     <widget class="QCheckBox" name="grRR">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>80</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>RR</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Individual</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Algorithm:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cache Size:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Queue Size:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="indAlg">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>LRU</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MRU</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RR</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="indCacheSize">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="indQueueSize">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="indLogOutputText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>631</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="indLogOutput">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>100</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="indDoAlg">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>100</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Do</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="About">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="alg"/>
    <addaction name="auth"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="About"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="alg">
   <property name="text">
    <string>About program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="auth">
   <property name="text">
    <string>About author</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
